### PR TITLE
Implemented tail similar to head

### PR DESCRIPTION
**Pull request by Nishchay-sopho**
This pull request is for the implementation of #1710. 

Please let me know if any other changes in this are required.

### CODE REVIEW

File: src/cmds/tail.c [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+	  int lines_checked = 0;  
+
+    unsigned long long position_of_last_line_to_read; 
+    char buf_str[2*SIZE]; 
+  
+    if (fseek(fp, 0, SEEK_END)) {
+        printf("fseek() failed"); 
+    	return ;	
+    }
+    else
```
Comment (anton-bondarev) [inline src/cmds/tail.c]: Why is else here? 'if' block has 'return' it's enough for me.

File: src/cmds/tail.c [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <errno.h>
+#include <ctype.h>
+
+#define SIZE 100
+
+static void read_last_n_lines(FILE *fp, int n){
+	  int lines_checked = 0;
```
Comment (anton-bondarev) [inline src/cmds/tail.c]: Please use Code-style use only tabs in a start line

File: src/cmds/tail.c [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+	int opt, option_number = 0, lines = 10, k;
+	int digit_checker = 1;	
+
+	if (argc < 2) {
+		printf ("Usage: more [FILE]\n");
+		return 0;
+	}
+
+	while (-1 != (opt = getopt(argc, argv, "n"))) {
+		// printf("opt= %d.\n",opt);
```
Comment (anton-bondarev) [inline src/cmds/tail.c]: remove it please

File: src/cmds/tail.c (hunk 2 above)
Comment (Nishchay-sopho) [reply src/cmds/tail.c]: Yeah, I am sorry. Will correct it and take special care of it from next time.

File: src/cmds/tail.c (hunk 1 above)
Comment (Nishchay-sopho) [reply src/cmds/tail.c]: Oh yeah, will change it right now.

File: src/cmds/tail.c (hunk 3 above)
Comment (Nishchay-sopho) [reply src/cmds/tail.c]: Done

**Comment by alexkalmuk** [comment]
@Nishchay-sopho How did you verify your changes do what expected?

**Comment by Nishchay-sopho** [comment]
I did the changes as asked by @anton-bondarev and was waiting for them to be reviewed. I am sorry, did I do something wrong ?

**Comment by Nishchay-sopho** [comment]
I have reverted the changes I had done which were solving the other issue #1698. Will take care that this doesn't happen again.

**Comment by Nishchay-sopho** [comment]
I have changed the file as per Code style. Could you please check and let me know ?

**Pull request closed without merging**